From 9eb06745126de85d032d2f5d65ea713065f03c3d Mon Sep 17 00:00:00 2001
From: Andrew Gregory <andrew.gregory.8@gmail.com>
Date: Sun, 17 Feb 2013 01:21:09 -0500
Subject: [PATCH] add SBINDIR variable to sys.mk

BINDIR was pointing to PREFIX/sbin which was confusing and inconsistent
with src/rc/Makefile.  Add SBINDIR and redefine BINDIR appropriately.

Signed-off-by: Andrew Gregory <andrew.gregory.8@gmail.com>
---
 mk/sys.mk | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/mk/sys.mk b/mk/sys.mk
index 251287f..ce9e8ea 100644
--- a/mk/sys.mk
+++ b/mk/sys.mk
@@ -29,9 +29,12 @@ CONFDIR?=		${SYSCONFDIR}/conf.d
 LOCALDIR?=		${SYSCONFDIR}/local.d
 SYSCTLDIR?=		${SYSCONFDIR}/sysctl.d
 
-BINDIR?=		${PREFIX}/sbin
+BINDIR?=		${PREFIX}/bin
 BINMODE?=		0755
 
+SBINDIR?=		${PREFIX}/sbin
+SBINMODE?=		0755
+
 INCDIR?=		${UPREFIX}/include
 INCMODE?=		0444
 
-- 
1.8.3

